libs/langchain-community/src/graphs/graph_document.ts

```diff
@@ -0,0 +1,53 @@
+import { Serializable } from "@langchain/core/load/serializable";
+import { Document } from "@langchain/core/documents";
+
+export class Node implements Serializable {
+    id: string | number;
+    type: string;
```

Comment:
`label: string` or `labels: string[]` ?

Reply:
Based on https://github.com/langchain-ai/langchain/blob/514fe807784f520449914a64ffc983538fa6743d/libs/community/langchain_community/graphs/graph_document.py#L20 it's just string.